ballerina_rest_asgardio_jwt/backend-server/server.bal

```diff
@@ -78,7 +78,7 @@ service /sales on new http:Listener(9090) {
 
     @http:ResourceConfig {
         auth: {
-            scopes: ["cargo_read"]
+            scopes: ["cargo_read", "cargo_insert"]
```

Comment:
Can't we use order_read only for this?
Admin can have both order_read and order_insert
Customers have order_read only